packages/guui/package.json

```diff
@@ -4,27 +4,43 @@
   "description": "Guardian rendering framework",
   "main": "index.js",
   "scripts": {
-    "test": "jest"
+    "test": "jest",
+    "prepublish": "cd ../.. && make pre-publish-guui",
+    "postpublish": "cd ../.. && make post-publish-guui"
+  },
+  "files": [
+    "document.js",
+    "grid.js",
+    "htmlTemplate.js",
+    "index.js"
+  ],
+  "directories": {
+    "lib": "./lib",
+    "components": "./components"
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "@guardian/guui": "2.0.0-alpha.1",
     "@guardian/pasteup": "0.1.0-alpha.1",
     "emotion": "^9.1.1",
     "emotion-server": "^9.1.1",
     "polished": "^1.9.2",
     "prop-types": "^15.6.1",
-    "react": "^16.4.0",
-    "react-dom": "^16.4.0",
-    "react-emotion": "^9.1.3",
     "reset-css": "^3.0.0"
   },
   "devDependencies": {
     "@babel/preset-react": "^7.0.0-beta.44",
     "babel-plugin-preval": "^1.6.4",
     "css-loader": "^0.28.11",
     "desvg-loader": "^0.1.0",
+    "react": "^16.4.0",
+    "react-dom": "^16.4.0",
+    "react-emotion": "^9.1.3",
     "svg-loader": "^0.0.2",
     "to-string-loader": "^1.1.5"
+  },
+  "peerDependencies": {
+    "react": "^16.4.0",
+    "react-dom": "^16.4.0",
+    "react-emotion": "^9.1.3"
```

Comment:
I'm not sure about this change. It would be better from a developer experience point of view if consumers of Guui didn't have to know about the libraries Guui consists of. What is the intention behind this? Just to allow consumers to bring their own versions of these libs?

Reply:
Good suggestion regarding the `index.js` entry point, I like that approach. Not sure whether this means we should be removing the `peerDependencies` (see my previous comment). I've put these here as initially I saw we were bundling these libraries with each component.  A situation which can lead to two versions of React be bundled in the consumers bundles. I put this line here https://github.com/guardian/dotcom-rendering/blob/gh-publish-guui/webpack/guui.js#L70 to prevent that. But this requires consumers of the library to requires consumers to install compatible versions themselves, hence the `peerDependencies`. 

What I haven't done which may be useful is to confirm 2 versions would be bundled. I'm just going on what I've read in many places (links are in the associated [Trello card](https://trello.com/c/oHF7fd6t/75-publish-guui-on-npm)).